connection.go

```diff
@@ -49,6 +49,50 @@ type ConnectionTransport interface {
 	Close()
 }
 
+type connTransport struct {
+	uri             *FMPURI
+	l               LogFactory
+	wef             WrapErrorFunc
+	conn            net.Conn
+	transport       Transporter
+	stagedTransport Transporter
+}
+
+var _ ConnectionTransport = (*connTransport)(nil)
```

Comment:
The API is the API. Changing it would require a bunch of changes in kbfs code that I don't know about.